exotica_examples/scripts/example_ik_look_at

```diff
@@ -26,7 +26,7 @@ class Example(object):
 
     def update(self, event):
         # Set new look at target into world frame
-        self.problem.get_scene().attach_object_local('LookAtTarget', '', self.target_marker.position_exo)
+        self.problem.get_scene().attach_object_local('LookAtTarget', '', self.target_marker.position_exo.get_translation_and_quaternion())
```

Comment:
So now we are converting KDLFrame => Eigen => KDLFrame ... is this really cleaner?

Reply:
In this example no, but often in scripts I find I am dealing with `np.array`s and it would be cleaner to pass in the `np.array` rather than `exo.KDLFrame(np.array([1,2,3]))` for example.

Reply:
With this change I only wanted to demonstrate that it works with the least amount of modifications to `exotica_examples`.